applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -391,28 +432,44 @@ def _CreateLevelSetConvectionProcess(self):
                 if domain_size == 2:
                     locator = KratosMultiphysics.BinBasedFastPointLocator2D(computing_model_part)
                     locator.UpdateSearchDatabase()
-                    level_set_convection_process = KratosConvDiff.BFECCConvection2D(locator)
+                    level_set_convection_process = KratosConvDiff.BFECCConvection2D(locator, self._levelset_splitting, True)
                 else:
                     locator = KratosMultiphysics.BinBasedFastPointLocator3D(computing_model_part)
                     locator.UpdateSearchDatabase()
-                    level_set_convection_process = KratosConvDiff.BFECCConvection3D(locator)
+                    level_set_convection_process = KratosConvDiff.BFECCConvection3D(locator, self._levelset_splitting, True)
             else:
                 raise Exception("The BFECC level set convection requires the Kratos ConvectionDiffusionApplication compilation.")
         else:
             linear_solver = self._GetLinearSolver()
+
+            levelset_convection_settings = self.settings["levelset_convection_settings"]
+
+            levelset_convection_settings["levelset_variable_name"].SetString("DISTANCE")
+            levelset_convection_settings["levelset_convection_variable_name"].SetString("VELOCITY")
+            levelset_convection_settings["levelset_gradient_variable_name"].SetString("DISTANCE_GRADIENT")
+
```

Comment:
```suggestion
```
These need to be set in the defaults (see my suggestion above).

Reply:
Done.